bl2/bl2_el3.ld.S

```diff
@@ -12,15 +12,26 @@ OUTPUT_ARCH(PLATFORM_LINKER_ARCH)
 ENTRY(bl2_entrypoint)
 
 MEMORY {
+#if BL2_AT_EL3_ROM
+    ROM (rx): ORIGIN = BL2_RO_BASE, LENGTH = BL2_RO_LIMIT - BL2_RO_BASE
+    RAM (rwx): ORIGIN = BL2_RW_BASE, LENGTH = BL2_RW_LIMIT - BL2_RW_BASE
```

Comment:
The porting guide will need updating to describe these new platform porting constants, and to say that these are only needed when BL2_IN_XIP_MEM is 1. Also, to say that BL2_LIMIT and BL2_BASE are not applicable in this case.

Reply:
Done, thanks.